tests/test_utils/test_schema_builder.py

```diff
@@ -0,0 +1,264 @@
+from typing import Optional, List, Union, Any, Dict
+from dataclasses import fields
+
+import pytest
+
+from linkml_runtime.utils.schema_builder import SchemaBuilder
+from linkml_runtime.linkml_model import (
+    ClassDefinition,
+    SlotDefinition,
+    EnumDefinition,
+    PermissibleValue,
+)
+
+
+# === Tests for `SchemaBuilder.add_class` ===
```

Comment:
any reason for the `# ===` comments - seems the docstrings should be sufficient?

Reply:
I used those to organize the tests into different sections, all tests for testing `SchemaBuilder.add_class`, all tests for testing `SchemaBuilder.add_enum`, and so on. I can organize the tests into classes like `class TestAddClass` and `class TestAddEnum`, but that will cost one indent. If you don't care about organizing the tests into different section, we can do it without the `# ===` comment and the class organization.

Reply:
let's stick to standard pytest function-based tests. I think it's fine, we have similar markers in other larger test files, at some point we should have a consistent style back not a blocker at all!

Reply:
Do you prefer to have the `# ===` lines removed?